packages/fungible-token/bridge-fungible-token/src/main.sw

```diff
@@ -197,27 +198,32 @@ impl Bridge for Contract {
     }
 }
 
-impl FRC20 for Contract {
+impl SRC20 for Contract {
     #[storage(read)]
-    fn total_supply() -> U256 {
-        let minted: u64 = storage.tokens_minted.try_read().unwrap_or(0);
+    fn total_assets() -> u64 {
```

Comment:
Why are the assets and total supply 0 / None?

Reply:
FIxed